etc/WMAgentConfig.py

```diff
@@ -356,3 +356,18 @@
 config.AgentStatusWatcher.userAMQ = "OVERWRITE_BY_SECRETS"
 config.AgentStatusWatcher.passAMQ = "OVERWRITE_BY_SECRETS"
 config.AgentStatusWatcher.topicAMQ = "OVERWRITE_BY_SECRETS"
+
+config.component_("RucioInjector")
+config.RucioInjector.namespace = "WMComponent.RucioInjector.RucioInjector"
+config.RucioInjector.componentDir = config.General.workDir + "/RucioInjector"
+config.RucioInjector.logLevel = globalLogLevel
+config.RucioInjector.enabled = False
+config.RucioInjector.pollInterval = 300
+config.RucioInjector.pollIntervalRules = 43200
+config.RucioInjector.cacheExpiration = 24 * 60 * 60  # one day
+config.RucioInjector.createBlockRules = True
+config.RucioInjector.useTestRSE = ""  # set it to "_Test" in a test node
```

Comment:
Maybe better to call it `.RSEPostfix` ?

Reply:
(Or just make it boolean with the postfix handled in code)

Reply:
I could change it, but I understand these Test RSEs are going away in a couple of weeks from now, no? @ericvaandering

Reply:
Nick, I just had a chat with Eric and apparently we have no use case for such _Test RSEs. I'm going to remove this code.